dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -143,12 +146,18 @@ cpu.processCpuLoad.description=Displays the 'recent cpu usage' for the Java Virt
 #Cpu processCpuTime message
 cpu.processCpuTime.description=Displays the CPU time used by the process on which the Java virtual machine is running.
 
+#Cpu processCpuUtilization message
+cpu.processCpuUtilization.description=The recent CPU time usage of the JVM process per processor available to the JVM.
```

Comment:
this one is confusing- not sure if my suggestion is correct. Time usage per available processor- is this an average of usage across all processors?

```suggestion
cpu.processCpuUtilization.description=The recent CPU time that is used by the JVM process for each processor that is available to the JVM.
```

Reply:
@dmuelle This metric calculates the recent cpu time that is used by the JVM process from all the processors that is available to the JVM. Would this sound better:
`The recent CPU time that is used by the JVM process from all processors that is available to the JVM.`

Reply:
yes- updated my suggestion

Reply:
Committed the change.